frontend/src/pages/modelRegistry/ModelRegistryRoutes.tsx

```diff
@@ -74,6 +76,26 @@ const ModelRegistryRoutes: React.FC = () => (
           </Route>
           <Route path="*" element={<Navigate to="." />} />
         </Route>
+        <Route path="registeredModels/archive">
+          <Route index element={<RegisteredModelsArchive empty={false} />} />
+          <Route path=":registeredModelId">
+            <Route index element={<Navigate to={ModelVersionsTab.DETAILS} />} />
```

Comment:
When the user clicks on the archive model in the table, they should be redirected to Versions tab instead of Details.
```suggestion
            <Route index element={<Navigate to={ModelVersionsTab.VERSIONS} />} />
```